ophyd/positioner.py

```diff
@@ -355,10 +355,6 @@ def read(self):
                         'timestamp': time.time()}
         return d
 
-    @property
```

Comment:
This, like `Signal` has a special value (enshrined as `self.position`). Therefore, I think it should keep this hint.